assets/src/edit-story/output/utils/getLongestMediaElement.js

```diff
@@ -27,9 +27,10 @@ import { getDefinitionForType } from '../../elements';
  */
 function getLongestMediaElement(elements) {
   return elements
-    .filter(({ type }) => {
+    .filter(({ type, loop }) => {
       const { isMedia } = getDefinitionForType(type);
-      return isMedia;
+      // Ensure looping media is not considered.
+      return isMedia && !loop;
```

Comment:
Should we also just discount images here or is that to support gifs (which can be non-looping too)?

Reply:
It could be used for audio in the future as well.